src/ripple/app/ledger/LedgerEntrySet.cpp

```diff
@@ -64,6 +69,7 @@ void LedgerEntrySet::swapWith (LedgerEntrySet& e)
     mSet.swap (e.mSet);
     std::swap (mParams, e.mParams);
     std::swap (mSeq, e.mSeq);
+    boost::swap (mFundingTable, e.mFundingTable);
```

Comment:
Should have been `std::swap` (or just `swap`, as Howard suggests). I must have just written a bunch of code that used `boost::` :-)